webapp/src/index.tsx

```diff
@@ -36,6 +30,12 @@ class Plugin {
     }
 }
 
+declare global {
+    interface Window {
+        registerPlugin(id: string, plugin: Plugin): void
+    }
+}
+
```

Comment:
I'm going to steal that one for https://github.com/mattermost/mattermost-plugin-starter-template :wink:

Reply:
"Imitation is the sincerest form of flattery" :smile: